index.html

```diff
@@ -4,9 +4,167 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./styles.css">
     <title>Douglas' Portifolio page</title>
 </head>
 <body>
-    <h2>Portifolio</h2>
+<div class="main-layout">
+
+    <nav>
+        <span>Luzinda Douglas</span> 
```

Comment:
Kindly wrap the logo in `a` tag 😃

- _This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info._